rpcserver.go

```diff
@@ -3499,19 +3499,17 @@ func handleGetCoinSupply(s *rpcServer, cmd interface{}, closeChan <-chan struct{
 		return s.coinSupplyTotal, nil
 	}
 
-	for i := 0; int64(i) < tipHeight+1; i++ {
-		if i == 0 {
-			continue
-		}
+	for i := 0; i < int(tipHeight/params.ReductionInterval); i++ {
```

Comment:
I believe this loop is unnecessary.  When counting begins the base subsidy should be the initial value (~31.28 DCR)